src/client/components/features/JeRecrute/DecouvrirDispositifs/DecouvrirDispositifs.test.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
+
+import { render, screen } from '@testing-library/react';
+
+import { DependenciesProvider } from '~/client/context/dependenciesContainer.context';
+
+import { DécouvrirDispositifs } from './DecouvrirDispositifs';
+
+describe('DecouvrirDispositifs', () => {
+
+	function renderComponent () {
+		render(
+			<DependenciesProvider>
+				<DécouvrirDispositifs />
+			</DependenciesProvider>,
+		);
+	}
+
+	describe('quand on clique sur Déposer une offre d‘emploi ou d‘alternance', () => {
+		it('ça te renvoie vers le formulaire du dépot de stage', () => {
```

Comment:
commence tout de suite l'intitulé de ton it avec `renvoie`, supprrime le `ça te`